newsfragments/295.fixed.md

```diff
@@ -0,0 +1 @@
+Reconfigure ethportal-peertest testing structure.
```

Comment:
```suggestion
Reconfigure ethportal-peertest testing structure to run multiple tests on a single spawned instance.
```

This is based on the PR description. Although I frankly have missed how this refactor makes it more possible than before. Could you explain a little more?.

Reply:
Ahh, not to run multiple tests on a single spawned instance, but multiple tests for a single jsonrpc endpoint (eg. `portal_stateFindContent`). Which was basically a result of how `validate_endpoint_response()` was built to match on a string representation of the endpoint itself.